react-frontend/src/recognition-list-components/RecognitionCardHeaderComponent.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { Row, Col } from "core-components/grid/GridComponent";
+import { Form } from "core-components/form/FormComponent";
+import { Button } from "core-components/button/ButtonComponent";
+import ImageComponent from "core-components/image/ImageComponent";
+
+const onclickMenuButton = () => {
+  //todo function for menu
+};
+
+const Img = styled.div`
+  width: 12vh;
+`;
+
+const RecognitionCardHeaderComponent = ({
+  given_at,
+  given_for,
+  core_value,
+}) => {
+  return (
+    <Row>
```

Comment:
I am not specifying the width this row occupies in container, neither I am using the col under it, do we need these?